src/identity/reducer.ts

```diff
@@ -55,7 +55,6 @@ export interface AddressToVerificationStatus {
 }
 
 interface State {
-  hasSeenVerificationNux: boolean
```

Comment:
This bit of state is fine to get rid of, since the only place we used it was the initial route logic anyways.

Reply:
🙏